app/controllers/master_files_controller.rb

```diff
@@ -42,7 +42,14 @@ def captions
   def waveform
     @master_file = MasterFile.find(params[:id])
     authorize! :read, @master_file
+
     ds = @master_file.waveform
```

Comment:
I don't think this is needed when generating an empty waveform (params[:empty] exists)